examples/cpp/route_guide/Makefile

```diff
@@ -86,7 +86,7 @@ ifneq ($(HAS_VALID_PROTOC),true)
 	@echo "Please install Google protocol buffers 3.0.0 and its compiler."
 	@echo "You can find it here:"
 	@echo
-	@echo "   https://github.com/google/protobuf/releases/tag/v3.0.0-alpha-1"
+	@echo "   https://github.com/google/protobuf/releases/tag/v3.0.0-beta-2"
```

Comment:
OK, added https://github.com/grpc/grpc/commit/fbf8c0b8ca235e606dcb914379825f6ff2691b7b to this PR